packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -0,0 +1,230 @@
+      maxWidth: '110px',
+      minWidth: '110px',
+    },
+    alignItems: 'center',
+  },
+  dialogueLeftContainerExpandedMobile: {
+    [theme.breakpoints.down('xs')]: {
+      marginBottom: '7px',
+    }
+  },
+  dialogueLeftContainerNoTopics: {
+    maxWidth: '200px',
+    minWidth: '200px',
+  },
+  dialogueMatchUsername: {
+    marginRight: 10,
+    '&&': {
+      color: theme.palette.text.primary,
+      textAlign: 'left'
+    },
+    whiteSpace: 'nowrap',
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    width: 'auto',
+    flexShrink: 'unset!important',
+  },
+  dialogueMatchCheckbox: {
+    marginLeft: 6,
+    width: 29,
+    '& label': {
+      marginRight: 0
+    }
+  },
+  debateTopicExpanded: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingBottom: '10px',
+    whiteSpace: 'normal',
+    [theme.breakpoints.down('xs')]: {
+      paddingLeft: '7px'
+    }
+  },
+  debateTopicCollapsed: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  },
+  reactIcon: {
+    paddingRight: "4px",
+  },
+  suggestionRow: {
+    display: 'flex',
+  },
+  agreeText: {
+    color: theme.palette.text.dim3,
+  },
+  agreeTextCollapsedMobile: {
+    [theme.breakpoints.down('xs')]: {
+      display: 'none'
+    },
+  },
+  topicRecommendationsList: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    color: theme.palette.text.primary,
+    fontSize: 'small',
+    overflow: 'hidden',
+    justifyContent: 'space-between'
+  },
+  expandIcon: {
+    cursor: 'pointer',
+    minWidth: '70px',
+    color: theme.palette.text.dim3,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    [theme.breakpoints.down('xs')]: {
+      marginLeft: 'auto',
+      minWidth: '25px',
+    },
+  },
+  hideIcon: {
+    cursor: 'pointer',
+    minWidth: '70px',
+    [theme.breakpoints.down('xs')]: {
+      marginLeft: 'auto',
+      minWidth: '25px',
+    },
+    color: theme.palette.text.dim3,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+  },
+  dialogueRightContainer: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    //paddingRight: 10,
+    //marginRight: 3,
+    marginLeft: 'auto',
+  },
+  dialogueMatchNote: {
+    [theme.breakpoints.down('xs')]: {
+      display: 'none'
+    }
+  },
+  bigScreenExpandNote: {
+    [theme.breakpoints.down('xs')]: {
+      display: 'none'
+    }
+  }
+});
+
+interface DialogueRecommendationRowProps {
+  rowProps: DialogueUserRowProps<boolean>; 
+  classes: ClassesType<typeof styles>; 
+  showSuggestedTopics: boolean;
+}
+
+const DialogueRecommendationRow = ({ rowProps, classes, showSuggestedTopics }: DialogueRecommendationRowProps) => {
+  const { DialogueCheckBox, UsersName, PostsItem2MetaInfo, LWTooltip, ReactionIcon } = Components
+
+  const { targetUser, checkId, userIsChecked, userIsMatched } = rowProps;
+  const { captureEvent } = useTracking(); //it is virtuous to add analytics tracking to new components
```

Comment:
I think it's only virtuous if you call `captureEvent`, which you don't appear to do